packages/devextreme-scss/scss/widgets/base/chat/layout/chat-messagelist/_mixins.scss

```diff
@@ -1,7 +1,49 @@
+@use "sass:math";
+
 @mixin chat-messagelist($padding) {
   .dx-chat-message-list {
     .dx-scrollable-content {
       padding: $padding;
     }
   }
 }
+
+@mixin chat-messagelist-empty(
+  $messagelist-empty-icon-box-size,
+  $messagelist-empty-icon-size,
+  $messagelist-empty-icon-margin-bottom,
+  $messagelist-empty-row-gap,
+  $messagelist-empty-icon-color,
+  $messagelist-empty-icon-background-color,
+  $messagelist-empty-empty-message-font-size,
```

Comment:
Repeat empty empty empty. Can we avoid it?

Reply:
Sure, it was a typo. Fixed https://github.com/DevExpress/DevExtreme/pull/28009/commits/8bb5b8a692e36490dbc343965a4ce2c4b5b61c88